File: home work/js/getListFromSpaceFile.js
const fs = require('fs');

const data = JSON.parse(fs.readFileSync('../spaces.json', 'utf8'));

data.folders.forEach(folder => {
    folder.lists.forEach(list => {
        console.log('List ID:', list.id);

        if (list.name.startsWith('test')) {
            console.log('Space Name:', list.space.name);
            console.log('Space ID:', list.space.id);
        }
    });
});

File: home work/python/getListFromSpaceFile.py
import json

with open('../spaces.json', 'r', encoding='utf-8') as file:
    data = json.load(file)

for folder in data['folders']:
    for lst in folder['lists']:
        print(f"List ID: {lst['id']}")

        if lst['name'].startswith('test'):
            print(f"Space Name: {lst['space']['name']}")
            print(f"Space ID: {lst['space']['id']}")